nginx.conf

```diff
@@ -0,0 +1,59 @@
+# User and worker settings
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+# Events block for handling connections
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+# HTTP block for handling HTTP requests
+http {
+    # Basic Settings
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    server_tokens off;
+
+    # Include MIME types
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    # SSL Settings
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+    ssl_prefer_server_ciphers on;
+
+    # Logging Settings
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/error.log;
+
+    # Gzip Settings
+    gzip on;
+    gzip_disable "msie6";
+
+    # Virtual Host Configs
+    include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/sites-enabled/*;
+
+    # Reverse proxy configuration
+    server {
+        listen 80;
+        # Server name can be domain or IP
+        server_name idrinth-api-ben.ch;
+
+        location / {
+            # Proxy pass to the Node.js application
+            proxy_pass http://localhost:8080;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection 'upgrade';
+            proxy_set_header Host $host;
+            proxy_cache_bypass $http_upgrade;
+        }
```

Comment:
what are you proxying to? We don't have a nodejs server?

Reply:
I read lots of articles and then wrote accordingly that I understood because I had never done it before. So, what can we use in there?

Reply:
The whole block has to be removed, it doesn't do anything for us. We are serving static files, not a nodejs server.